openspec/specs/opsx-archive-skill/spec.md

```
@@ -0,0 +1,122 @@
+# OPSX Archive Skill Spec
+
+### Requirement: OPSX Archive Skill
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)